### PR TITLE
ENCD-4593 Split ChIP-seq

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

File: src/encoded/schemas/changelogs/target.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for target.json
 
+### Minor changes since schema version 10
+
+* *investigated_as* enum changed to delete "control", "nucleotide modification", "other post-translational modification", and to add "Co-factor", "Cohesin", "DNA replication", "DNA repair", "RNA polymerase complex".
```
Comment (gabdank) [inline src/encoded/schemas/changelogs/target.md]: I think that should be slightly reworded to be: "enum list was modified. The following terms were removed .. and the following terms were added ..."

Comment (gabdank) [reply src/encoded/schemas/changelogs/target.md]: And "control" was not deleted from the list

Comment (yunhailuo) [reply src/encoded/schemas/changelogs/target.md]: Fixed with the new commit.

**Comment by yunhailuo** [comment]
> @yunhailuo ENCD-3108 has to be merged before this PR?

Yes, that's the plan.

**Comment by yunhailuo** [comment]
> @yunhailuo can you please add a test that checks the proper choice of preffered_name in ChIP experiment?

Added. Can you check if that's good enough, @gabdank ?

**Comment by yunhailuo** [comment]
This is a "Multi-Jira Ticket Pull Requests". #2742 should get in first, @caseylitton .

File: src/encoded/schemas/changelogs/target.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
 ## Changelog for target.json
 
+### Schema version 11
```
Comment (caseylitton) [inline src/encoded/schemas/changelogs/target.md]: flagging for removal of revert commit

**Pull request merged**